easybuild/easyconfigs/n/ncregrid/ncregrid-1.4b-goolf-1.4.10.eb

```diff
@@ -0,0 +1,37 @@
+##This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2014 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+#
+##name = 'ncregrid'
+
+version = '1.4b'
+
+homepage = 'http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/'
+description = """ncregrid is a tool (written in FORTRAN-90) for data transfer of gridded 2- and 3-dimensional (spatial) geophysical/geochemical scalar fields between grids of different resolutions"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'usempi': True, 'pic': True}
+
+source_urls = ['http://www.pa.op.dlr.de/~PatrickJoeckel/ncregrid/']
+sources = [('%(namelower)s.tar.Z', 'tar -xzf %s')]
```

Comment:
As hpcugent/easybuild-framework#1396 is merged, this can be replaced by `SOURCELOWER_TAR_Z`